examples/notebook/dynas/Supernet_NAS.ipynb

```diff
@@ -13,7 +13,7 @@
     "\n",
```

Comment:
May be better to use separate notebook for different models?

Reply:
Done!